### DISCUSSION TITLE
How to save Bootstrap components as self-sufficient code?

### DISCUSSION THREAD

**Discussion by e-orlov** [twbs/bootstrap#37066] [Q&A] [answered]
I need only one Bootstrap component. HTML code is always displayed in the documentation. But not CSS and JS - I'm forced to implement both of `bootstrap.min.css` and `bootstrap.bundle.min.js` in my site. I feel both files as a huge overhead and want to avoid. 

How can I save a Bootstrap component as HTML with only corresponding with it CSS rules and JS functions, without to load main CSS and JS files?

Anything like this, or even with inline style.
```
<script>...</script>
<style>...</style>
<div class="accordion">...</div>
```

**Comment by mdo** [comment, marked as answer]
If you need only a fraction of what Bootstrap offers, your best bet is looking at a bundler and creating your own custom builds. This is demonstrated in a number of JS frameworks and tools in our https://github.com/twbs/examples repo.
